### DISCUSSION TITLE
How do I hyperlink a table entry?

### DISCUSSION THREAD

**Discussion by hoffenmerCode** [beeware/toga#1273] [Q&A]
I have a simple table with entries. One of the entries is an https:// hyperlink. How do I make it 'clickable' in toga?

**Comment by freakboy3742** [comment]
Moving this to a discussion since it's a question, rather than a feature enhancement. 

At present, we don't have support for hyperlink content in Toga (in tables, or anywhere else). The idea has been raised in the context of Labels (#801); it would make sense that whatever approach is used there would also work in Table content as well (or, at least, an analogous/compatible implementation)
